core/jazz_t-vault/components/utils/vault.js

```diff
@@ -103,7 +104,7 @@ function getSafeDetails(safename, configData, vaultToken, onComplete) {
     } else {
       logger.error("Error in getting safe details: " + JSON.stringify(error));
       return onComplete({
-        "error": "Error in getting safe details. " + response.body
+        "error": `Error in getting safe details with safe name ${safename} : ${response.body}`
```

Comment:
Consistently no spaces before colon (across this api), should be `with safe name ${safename}:`